packages/web-components/src/components/va-accordion/va-accordion-item.e2e.ts

```diff
@@ -150,10 +150,24 @@ describe('va-accordion-item', () => {
     await page.setContent(
       '<va-accordion-item subheader="The subheader"><h3 slot="headline">The header</h3>Content inside</va-accordion-item>',
     );
-    
+
     const element = await page.find('va-accordion-item');
     let subheader = element.shadowRoot.querySelector('p');
 
     expect(subheader).toEqualText('The subheader');
   });
+
+  it.skip('fires a custom event when the location hash matches the accordion id', async () => {
+    const page = await newE2EPage();
+
+    // await page.evaluate(() => { window.location.hash = '#testing'; });
+    await page.goto(`/${page.url()}#testing`);
+
+    const accordionItemToggled = await page.spyOnEvent('accordionItemToggled');
+    await page.setContent(
+      '<va-accordion-item id="testing" header="The header">Content inside</va-accordion-item>',
+    );
+
+    expect(accordionItemToggled).toHaveReceivedEventTimes(1);
+  });
```

Comment:
I need help figuring out why this hash change isn't working

Reply:
I wasn't able to get it to work using page.goto, but we can specify the url when creating the page like this:

```
    const page = await newE2EPage({ url: '/#testing' });
    await page.setContent(
      '<va-accordion-item id="testing" header="The header">Content inside</va-accordion-item>',
    );
    const accordionItemToggled = await page.spyOnEvent('accordionItemToggled');
    await page.waitForChanges();
    expect(accordionItemToggled).toHaveReceivedEvent();
```

This simplifies the test by removing navigation and focuses on `va-accordion-item`'s componentDidLoad behavior when its condition is true.

Reply:
Thanks, that worked much better!